staff/henry-novoa/precourse/tema1/exercises1.js

```diff
@@ -0,0 +1,203 @@
+
+//Strings
+//a) Puedes contar cuantas letras tiene tu nombre?
+
+var myName = "Henry Novoa";
+var myAge = 28;
+var myCity = "Barcelona";
+function showLetters(name){
+	var length = name.length - 1; 
+	console.log("My name has " + length + " letters");
+}
+showLetters(myName);
+//b) Añade tu apellido e indica en que posición del string empieza (prueba a buscar el espacio entre el nombre y el apellido):
+function space(name) {
+	var position = 0;
+
+	/*for (var i = 0; i < name.length ; i++){
+		if(name[i] === " "){
+			position = i +1;
+		}
+	}*/
+	position=name.indexOf(" ") + 1;
+
+	console.log("The last name starts in the "+position+" character");
+
+}space(myName)
```

Comment:
separate function call to another line, avoid putting it all inline (and don't forget the semicolon):

}

space(myName);